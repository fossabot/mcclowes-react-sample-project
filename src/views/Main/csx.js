import styled from "styled-components";

export const Page = styled("div")`
  align-items: center;
  background: rgb(215, 13, 107);
  background: linear-gradient(
    52deg,
    rgba(215, 13, 107, 1) 0%,
    rgba(50, 63, 123, 1) 100%
  );
  display: flex;
  height: 100vh;
  justify-content: center;
  overflow: hidden;
  width: 100vw;
`;
