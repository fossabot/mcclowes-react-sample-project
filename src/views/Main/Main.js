import Dumb from "../../components/Dumb";
import React from "react";
import Smart from "../../components/Smart";
import { Page, PageWrapper } from "./csx";
import { Text } from "../../components/UIKit";
import Anime from "react-anime";

const Main = (props) => {
  return (
    <PageWrapper>
      <Anime
        opacity={[0, 1]}
        translateY={["-4em", 0]}
        delay={(e, i) => i * 2000}
      >
        <Page>
          <Smart>
            <Text>Hello World!</Text>
          </Smart>

          <Dumb>
            <Text>
              This is an example project. Make changes as you see fit.
            </Text>
          </Dumb>
        </Page>
      </Anime>
    </PageWrapper>
  );
};

export default Main;
