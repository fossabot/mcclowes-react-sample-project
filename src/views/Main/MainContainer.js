import React, { useState } from "react";
import Main from "./Main";

const MainContainer = props => {
  const [loading, setloading] = useState(false);

  return <Main {...props} />;
};

export default MainContainer;
