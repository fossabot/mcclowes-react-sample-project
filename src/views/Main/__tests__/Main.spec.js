import Main from "../Main";
import React from "react";
import { shallow } from "enzyme";

it("renders component", () => {
  const wrapper = shallow(<Main />);

  expect(wrapper).toMatchSnapshot();
});
