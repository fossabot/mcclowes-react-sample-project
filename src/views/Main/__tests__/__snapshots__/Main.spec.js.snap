// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<csx__PageWrapper>
  <e
    delay={[Function]}
    opacity={
      Array [
        0,
        1,
      ]
    }
    translateY={
      Array [
        "-4em",
        0,
      ]
    }
  >
    <csx__Page>
      <SmartContainer>
        <Text>
          Hello World!
        </Text>
      </SmartContainer>
      <Dumb>
        <Text>
          This is an example project. Make changes as you see fit.
        </Text>
      </Dumb>
    </csx__Page>
  </e>
</csx__PageWrapper>
`;
